tt_metal/hw/inc/grayskull/dev_mem_map.h

```diff
@@ -103,7 +103,7 @@
 /////////////
 // Padding/alignment restrictions needed in linker scripts
 // Note we pad then align so, for example, erisc can keep space between fw/kernels
```

Comment:
and these comments too? padding of zero is not interesting

Reply:
yeah, good catch